src/events/http2.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * @fileoverview Handlers for http & https libraries instrumentation
+ */
+
+const uuid4 = require('uuid4');
+const shimmer = require('shimmer');
+const urlLib = require('url');
+const http2 = require('http2');
```

Comment:
maybe should be tryRequire (not supported in older versions)

Reply:
HTTP2 appears since node8. I think we will just remove the support for node6 because that's very old

Reply:
Fixed